### ISSUE TITLE
Focus order of Expand Sum button should be just after Sum ExpandCollapseButton

### ISSUE THREAD

**Issue by arouinfar** [phetsims/fourier-making-waves#189]
For #178 

The focus order of the button that opens the dialog containing the expanded form of the sum is currently after the ResetAllButton.
![image](https://user-images.githubusercontent.com/8419308/134995186-d9449ce2-f3bc-46bb-b0c5-fccdf5ee6abc.png)

I think it would make more sense for it to come between the Sum ExpandCollapseButton and the Sum graph's x-axis zoom buttons since that's where it visually appears.

**Comment by arouinfar** [comment]
👍🏻
